### PR TITLE
Lint code

### PR DESCRIPTION
**Pull request by DengYiping**

### CODE REVIEW

File: app/js/components/hello.all.js [hunk 1]
```diff
@@ -3274,7 +3274,7 @@ hello.utils.extend(hello.utils, {
 			xhr: false,
 
 			jsonp: function(p, qs) {
-				if (p.method == 'get') {
+				if (p.method === 'get') {
```
Comment (sudheesh001) [inline app/js/components/hello.all.js]: This ideally has to go upstream to the hello.js library as a patch.

File: app/js/components/highlight.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 !function(e) {
-    "undefined" != typeof exports ? e(exports) : (window.hljs = e({}),
-    "function" == typeof define && define.amd && define("hljs", [], function() {
+    "undefined" !== typeof exports ? e(exports) : (window.hljs = e({}),
+    "function" === typeof define && define.amd && define("hljs", [], function() {
```
Comment (sudheesh001) [inline app/js/components/highlight.js]: Similarly this should go to highlight.js as a patch. Maybe we need to leave `js/components` out of linting.

**Comment by DengYiping** [comment]
OK,leave ```js/componets``` alone.

**Comment by DengYiping** [comment]
@sudheesh001  Should be good. I download those two files and replace the one I changed.

**Comment by sudheesh001** [comment]
@DengYiping Sorry for the late response, I like the changes made but there are still 2 files in the js/components which have been made, Could you remove those files, stash and push the commits ?

**Comment by DengYiping** [comment]
OK, I will make the change.

**Comment by DengYiping** [comment]
@sudheesh001  Should be all good.

**Pull request merged**